doc/active_docs/account_management_api.json

```diff
@@ -5879,19 +5878,14 @@
                     "type": "string",
                     "description": "System Name of the metric. If blank a system_name will be generated for you from the friendly_name parameter."
                   },
-                  "unit": {
```

Comment:
Can you also please also make the following changes:
- change "metric" to "method" in all descriptions of the request body (but not "metric_id" path param, because here it must is about metric)
- while on it, I'd change all mentions of "api" (lowercase) in the params decriptions to "API".

Reply:
then 

Creates a method under a metric that belongs to a backend API.

need to be changed to 
reates a method  that belongs to a backend API.

Reply:
No, I meant:

"Descriptive Name of the metric." => "Descriptive name of the method."

"System Name of the metric. If blank a system_name will be generated for you from the friendly_name parameter." => "System name of the method. If not specified, the system name will be generated from the value of the `friendly_name` parameter."

(here the main point was changing "metric" to "method", but I reworded it, because I think the description was quite bad)

"Description of the metric." => "Description of the method."

The same for Create and Update calls.

Reply:
Also, I noticed that "Service Method Create" and "Service Method Update" have the same issues ("unit" field that shouldn't be there, and "metric" in the params descriptions which should be changed to "method").

Can you please change that also?

Reply:
sure